src/components/Artifacts/Artifact.vue

```diff
@@ -20,10 +20,19 @@ export default {
 
 <template>
   <div
+    v-if="artifact.kind == 'md'"
```

Comment:
This isn't catching markdown in [some flows](https://deploy-preview-559--prefect-ui.netlify.app/prefect/flow-run/595ac283-9c66-4241-8edd-4b21733b95f7?artifacts=).  Should it be "artifact.kind == markdown"?

Reply:
Ah, you're right - I've added that!